### PR TITLE
WL-4020 Applied patch for Lessons announcements widget from WL-10 to 11.

### PR DESCRIPTION
**Pull request by ouit0408**
Changes from announcement entity provider are added in the commit.

### CODE REVIEW

**Comment by buckett** [comment]
Sorry, doesn't merge now.

**Comment by ouit0408** [comment]
rebased

File: announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/entityprovider/AnnouncementEntityProviderImpl.java [hunk 1]
```diff
@@ -224,6 +225,17 @@ public String getEntityPrefix() {
 				announcements.addAll(announcementService.getMessages(channel, t, numberOfAnnouncements, true, false, onlyPublic));
 			} catch (PermissionException e) {
 				log.warn("User: " + currentUserId + " does not have access to view the announcement channel: " + channel + ". Skipping...");
```
Comment (buckett) [inline announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/entityprovider/AnnouncementEntityProviderImpl.java]: We shouldn't have a warning in an expected code path.

Comment (ouit0408) [inline announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/entityprovider/AnnouncementEntityProviderImpl.java]: it was present already in the code, shall i change it?

Comment (buckett) [inline announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/entityprovider/AnnouncementEntityProviderImpl.java]: Yes as the message is also wrong, it's not going to skip it.

Comment (ouit0408) [inline announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/entityprovider/AnnouncementEntityProviderImpl.java]: made the change.

**Pull request merged**